apps/web/pages/auth/setup.tsx

```diff
@@ -0,0 +1,293 @@
+import { CheckIcon } from "@heroicons/react/solid";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useRouter } from "next/router";
+import { Controller, useForm } from "react-hook-form";
+import * as z from "zod";
+
+import { isPasswordValid } from "@calcom/lib/auth";
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { inferSSRProps } from "@calcom/types/inferSSRProps";
+import WizardForm from "@calcom/ui/WizardForm";
```

Comment:
@leog cc @sean-brydon maybe we want to put this in `/v2/` for now. Also, is this the same that we have on console?

Reply:
Yes, is the same, moved it from `console` to here and added with a few tweaks. Sure, I can move it to `v2` along the other components.

Reply:
@zomars after chatting with @sean-brydon it seems like we can't move these components to `v2` as there is a color scheme mismatch between that and web where these components are being used. So, for now we should stay put.

Reply:
Let's update these components to match the new color scheme. This shouldn't be a blocker